go/libraries/doltcore/schema/typeinfo/decimal.go

```diff
@@ -37,6 +37,8 @@ type decimalType struct {
 
 var _ TypeInfo = (*decimalType)(nil)
 
+var DecimalDefaultType = &decimalType{sql.MustCreateDecimalType(10, 5)}
```

Comment:
I'm not sure that we should declare this as the "default type", as I don't really think that there should be one for the decimal type. Any place that wants to construct a decimal type as a default value should declare it at the call site IMO.